js/jquery.terminal.js

```diff
@@ -1357,6 +1358,8 @@
                     } else {
                         if (command !== '' && position > 0) {
                             self['delete'](-1);
+                            var str = $('.clipboard').val()
+                            $('.clipboard').val(str.substring(0, str.length - 1));
```

Comment:
I don't expect you to make those changes, just thinking out loud for improvements.